Program.cs

```diff
@@ -1,4 +1,26 @@
-﻿using System;
-Console.WriteLine ("Hello, World!");
-
-   
\ No newline at end of file
+﻿// --------------------------------------------------------------------------------------------
+// Training ~ A training program for interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.                                              
+// ------------------------------------------------------------------------
+// Program.cs
+// Extension of Spelling Bee assignment
+// Display the first 7 letters to be used as the seed for the Spelling Bee program from the given word list.
+// --------------------------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+// Reading all letter in the wordlist.
+string wordsList = File.ReadAllText ("D:\\OneDrive - Trumpf Metamation Pvt Ltd\\words.txt");
+Dictionary<char, int> output = new ();
+foreach (char ch in wordsList) {
+   if (ch >= 'A' && ch <= 'Z') {
+      // Initialize count to 1 for a new character
+      if (!output.ContainsKey (ch)) output[ch] = 1;
+      // Increment count for an existing character
+      else output[ch]++;

```

Comment:
The ideal way to update a dictionary is `output.TryGetValue (ch, out int cnt);
      output[ch] = cnt + 1;` In your code, when the character is already present in the dictionary, it will actually do the dictionary search 3 times. Once during the `ContainsKey` call, and twice during the `++` (since that is a read followed by a write). In contrast, the solution with `TryGetValue` will access the dictionary only twice. Also, it does not use any if statements.